### ISSUE TITLE
rats fail with cf cli 6.33.1

### ISSUE THREAD

**Issue by kkallday** [cloudfoundry/routing-release#99]
## Issue

We noticed that rats is failing with the latest cf cli 6.33.1.

## Context

```
• Failure in Spec Setup (BeforeEach) [2.233 seconds]
Tcp Routing
/tmp/build/ddc81735/acceptance-tests/src/code.cloudfoundry.org/routing-acceptance-tests/tcp_routing/tcp_routing_test.go:19
  single app port
  /tmp/build/ddc81735/acceptance-tests/src/code.cloudfoundry.org/routing-acceptance-tests/tcp_routing/tcp_routing_test.go:24
    when multiple external ports are mapped to a single app port [BeforeEach]
    /tmp/build/ddc81735/acceptance-tests/src/code.cloudfoundry.org/routing-acceptance-tests/tcp_routing/tcp_routing_test.go:112
      routes traffic from two external ports to the app
      /tmp/build/ddc81735/acceptance-tests/src/code.cloudfoundry.org/routing-acceptance-tests/tcp_routing/tcp_routing_test.go:122

      Expected
          <int>: 0
      to equal
          <int>: 2

      /tmp/build/ddc81735/acceptance-tests/src/code.cloudfoundry.org/cf-routing-test-helpers/helpers/routes.go:65
```
The assertion line is: https://github.com/cloudfoundry/cf-routing-test-helpers/blob/9b3553a85dfb2662ac94f78cab32d801009d8f3c/helpers/routes.go#L65

Our previous run with cf cli 6.33.0 passes. We believe this is due to a change in the cf cli where they are printing stdout response of the `create route` command with ANSI colors. The regex is now broken.

This might be the CF CLI story that introduced the issue: https://www.pivotaltracker.com/n/projects/892938/stories/153200724

## Steps to Reproduce

1. Run RATs with CF CLI 6.33.1

## Expected result

RATs pass.

## Current result

RATs fail.

## Possible Fix

Not a fix but it might be a good idea to also escape the special chars in the domain when building the regex.

cc @acrmp

**Comment by charleshansen** [comment]
Tested this locally, the new output with color looks like
`Route ^[[36;1mDOMAIN:PORT^[[0m has been created.`

**Comment by shubhaat** [comment]
We fixed this issue with routing-release 0.170.0
